Neos.EventSourcedContentRepository/Classes/Domain/Context/NodeAggregate/NodeAggregate.php

```diff
@@ -182,6 +448,17 @@ public function getStreamName(): StreamName
         return $this->streamName;
     }
 
+    private function traverseEventStream(callable $callback): void
```

Comment:
maybe make this return a Generator? then it's easier to use and can be made more efficient.

Reply:
I'm not sure. If it improves the API, sure, but I don't think it can provide any efficiency improvements here.

Reply:
was a nitpick from my side :)